### PR TITLE
Update some more dependencies.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: dlp/api/DlpSample/DeIdentify.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@
 using System.IO;
 using static Google.Cloud.Dlp.V2.CryptoReplaceFfxFpeConfig.Types;
 using System.Linq;
+using Google.Api.Gax.ResourceNames;
```
Comment (simonz130) [inline dlp/api/DlpSample/DeIdentify.cs]: sort this

File: dlp/api/DlpSample/RiskAnalysis.cs [hunk 2]
```diff
@@ -74,14 +75,14 @@ class RiskAnalysis
...
-                    ParentAsProjectName = new Google.Cloud.Dlp.V2.ProjectName(callingProjectId),
+                    ParentAsProjectName = new ProjectName(callingProjectId),
                     RiskJob = config
                 });
 
             // Listen to pub/sub for the job
             var subscriptionName = new SubscriptionName(callingProjectId, subscriptionId);
-            SubscriberClient subscriber = SubscriberClient.Create(
-                subscriptionName, new[] { SubscriberServiceApiClient.Create() });
+            SubscriberClient subscriber = SubscriberClient.CreateAsync(
```
Comment (simonz130) [inline dlp/api/DlpSample/RiskAnalysis.cs]: Did they change the API? Why did you change the original code? It's better to call sync method directly than call async method and then .Result

File: dlp/api/DlpSample/DeIdentify.cs (hunk 1 above)
Comment (SurferJeffAtGoogle) [reply dlp/api/DlpSample/DeIdentify.cs]: Done.

File: dlp/api/DlpSample/RiskAnalysis.cs (hunk 2 above)
Comment (SurferJeffAtGoogle) [reply dlp/api/DlpSample/RiskAnalysis.cs]: Yes, they changed the API.  I know it's better to await the result, but that would require making all, or most of the functions in the call stack async, which is overkill for this sample that otherwise does nothing asynchronous.
